packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -385,6 +413,12 @@ export default class ReconnectionManager {
       'ReconnectionManager:index#executeReconnection --> Attempting to reconnect to meeting.'
     );
 
+    const wasSharing = this.meeting.shareStatus === SHARE_STATUS.LOCAL_SHARE_ACTIVE;
+
+    if (wasSharing) {
```

Comment:
Share is going to stop on network change also this time , can you check

Reply:
Share will not stop unless the peer connection state goes to failed.